src/components/chat-sidebar/supervisor-sidebar-link.tsx

```diff
@@ -15,12 +17,16 @@ export function SupervisorSidebarLink({
   prompt: string;
   removeSupervisor: (_uuid: string) => void;
 }>) {
+  const pathname = usePathname();
   return (
     <SidebarMenuItem className="rounded-md border-2 border-sidebar-accent">
       <SidebarMenuButton asChild>
         <Link
           href={`/supervisor/${uuid}`}
-          className="flex h-fit flex-row items-center justify-between"
+          className={cn(
+            "flex h-fit flex-row items-center justify-between",
+            pathname.includes(uuid) ? "bg-chat-user" : "",
+          )}
         >
```

Comment:
```suggestion
    <SidebarMenuItem
      className={cn(
        "rounded-md border-2 border-sidebar-accent",
        pathname.includes(uuid) ? "bg-chat-user" : "",
      )}
    >
      <SidebarMenuButton asChild>
        <Link
          href={`/supervisor/${uuid}`}
          className={cn("flex h-fit flex-row items-center justify-between")}
        >
```
I would apply the background on `SidebarMenuItem` instead of the `Link` because there's a very small, but noticeable gap 

![image](https://github.com/user-attachments/assets/f3c3b0ae-3312-4f35-8c29-c553c4ae38b0)